scripts/clean-db.py

```diff
@@ -0,0 +1,26 @@
+import argparse
+import os
+import sys
+
+sys.path.insert(1, "/".join(os.path.realpath(__file__).split("/")[0:-2]))
+
+from harvester import HarvesterDBInterface  # noqa E402
+
+
+def main(days):
+    interface = HarvesterDBInterface()
+    outdated_records = interface.get_all_outdated_records(days)
+    for record in outdated_records:
+        interface.delete_harvest_record(ID=record.id)  # this cascades to errors
```

Comment:
thanks for the comment. this will change once we hang errors on the job instead of the record.